### PR TITLE
Add `-loader` to webpack loaders

### PR DESCRIPTION
**Pull request by Matt-Butler**
### Summary
During development, I was only testing with webpack 1. Going through with webpack 2, I ran into the issue where you can no longer omit `-loader` from webpack loaders. 

`BREAKING CHANGE: It's no longer allowed to omit the '-loader' suffix when using loaders.
                 You need to specify 'babel-loader' instead of 'babel'.`

After adding loader to this file, the terra-react-svg-loader is compatible with webpack 2.

https://github.com/cerner/terra-icon/issues/12

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -78,18 +78,18 @@
...
-    "lint": "yarn lint-js",
+    "lint": "npm run lint-js",
     "lint-js": "eslint --ext .js,.jsx .",
-    "pretest": "yarn lint",
-    "release:major": "yarn test && yarn build && npm version major -m \"Released version %s\" && npm publish && git push --follow-tags",
-    "release:minor": "yarn test && yarn build && npm version minor -m \"Released version %s\" && npm publish && git push --follow-tags",
-    "release:patch": "yarn test && yarn build && npm version patch -m \"Released version %s\" && npm publish && git push --follow-tags",
+    "pretest": "npm run  lint",
+    "release:major": "npm run  test && npm run  build && npm version major -m \"Released version %s\" && npm publish && git push --follow-tags",
+    "release:minor": "yarn test && npm run  build && npm version minor -m \"Released version %s\" && npm publish && git push --follow-tags",
```
Comment (StephenEsser) [inline package.json]: Missed one yarn reference

Comment (Matt-Butler) [inline package.json]: good catch. fixed

**Comment by Matt-Butler** [comment]
This thread had the same yarn error that I was receiving: https://github.com/yarnpkg/yarn/issues/2428#issuecomment-274706284

Switching back to npm fixed the issue.

**Comment by mhemesath** [comment]
👍

**Pull request merged**